dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -57,72 +58,117 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
 
     void AppNotificationManager::Register()
     {
-        winrt::guid storedComActivatorGuid{ GUID_NULL };
-        if (!PushNotificationHelpers::IsPackagedAppScenario())
+        HRESULT hr{ S_OK };
+        std::wstring appId{};
+
+        auto logTelemetry{ wil::scope_exit([&]() {
+            AppNotificationTelemetry::RegisterByAPI(hr, appId);
```

Comment:
Unless I'm missing something critical here, appId is always empty.

Reply:
It's possible that there are a few functions that weren't using the appid internally, so I can either remove it from the corresponding telemetry events or fetch it in the function so it's part of the telemetry event. The telemetry events already log the app name.

Reply:
Logging appName based on what? ProcessName?

Reply:
Please remove appId if you believe it isn't critical to logging. Totally fine with that as part of V1 logging story since we use ProcessName/AppUserModelId as AppName instead.

Reply:
All usage of appid have been addressed